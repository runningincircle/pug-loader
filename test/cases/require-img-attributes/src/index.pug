p Hello World!

img&attributes({
  class: 'image',
  src: require('./image.png'),
  srcset: `${require('./image1.png')} 80w, ${require('./image2.png')} 90w`,
  alt: 'photo',
})