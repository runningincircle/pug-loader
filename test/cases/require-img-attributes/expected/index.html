<p>Hello World!</p><img class="image" src="/assets/images/image.697ef306.png" srcset="/assets/images/image1.697ef306.png 80w, /assets/images/image2.697ef306.png 90w" alt="photo">